api/db/populate-users.js

```diff
@@ -139,6 +141,10 @@ const createUsers = async (multiplier, randomLocation, verbose) => {
   }
 
   console.log('\x1b[32m', `Created ${usersArray.length} regular users`);
+  console.log(
+    '\x1b[32m',
+    `Created ${notificationsCount} notifications related to creation of a new user`
```

Comment:
It feels a bit weird to me that we include here the number of notifications related to the admin user but just above only count the regular users we have created :confused:

Reply:
Easiest solution would be to include the admin in the count. It's kind of irrelevant whether or not to count admin in the logging.

Reply:
Yep, let's just do that (removing the `regular` word of the sentence)!